src/repos/NotificationRepo.ts

```diff
@@ -0,0 +1,139 @@
+import apn from 'apn';
+import * as admin from 'firebase-admin';
+import { User } from '../entities/User';
+import { Publication } from '../entities/Publication';
+import { Article } from '../entities/Article';
+import UserRepo from './UserRepo';
+import PublicationRepo from './PublicationRepo';
+import ArticleRepo from './ArticleRepo';
+import { ANDROID, IOS } from '../common/constants';
+
+/**
+ * Send IOS push notification to {userID} about an {articleID} that {publicationSlug} posted.
+ */
+const sendIOSNotification = async (
+  user: User,
+  article: Article,
+  publication: Publication,
+): Promise<void> => {
+  // console.log('Sending IOS Notification...');
+  // Get device token of user
+  const { deviceToken } = user;
```

Comment:
Do you ever verify that this user is an iOS user? Not sure if that's necessary but might be a good thing to double-check

Reply:
wait i saw down below that you only call this on ios users so its prob unnecessary to double check haha